node/plain/webappCosmosDB/Application/Web.Debug.config

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- This file will modify Web.config to enable remote debugging when publishing your project in the Debug configuration.
+     For more information on using web.config transformation visit http://go.microsoft.com/fwlink/?LinkId=125889 -->
+<configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+
+  <system.web xdt:Transform="Insert">
+    <httpRuntime targetFramework="4.5" />
+    <customErrors mode="Off"/>
+  </system.web>
+
+  <system.webServer>
+    <!-- mimeMap enables IIS to serve particular file types as specified by fileExtension. -->
+    <staticContent>
+      <mimeMap fileExtension=".svg" mimeType="image/svg+xml" />
+    </staticContent>
+    
+    <iisnode loggingEnabled="true"
```

Comment:
You should use tis file from the node express and node plain projects in this repository.